common/components/ConfirmationModalTemplate/components/Body/components/Addresses.scss

```diff
@@ -0,0 +1,67 @@
+@import 'common/sass/variables';
+
+.tx-modal-address {
+  display: flex;
+  flex-direction: column;
+  width: fit-content;
```

Comment:
The support for this isn't perfect. However the support for just `width: fit-content` is pretty good. The only exception is ie & edge. Assuming we aim to also support IE, together they total 4.34%, IE – 2.84%, Edge – 1.5%.

In this case achieving the same alignment without `width: fit-content;` would be difficult and verbose. I'd suggest instead of scrapping `fit-content` we fallback to another with property. And with the cascading effect of css it's super easy
```
width: 100%;
width: fit-content;
```

Reply:
nvm, went back and double checked, I don't actually need it.